src/java.base/share/classes/jdk/internal/classfile/impl/TargetInfoImpl.java

```diff
@@ -31,7 +31,7 @@
 import static jdk.internal.classfile.Classfile.*;
 
 /**
- *
+ * TargetInfoImpl
```

Comment:
Does this javadoc add any value?

Reply:
This is common practise across the whole implementation. Do you suggest to remove all similar javadoc from all implementation classes?

Reply:
Well, if the javadoc simply states the name of the class it doesn't seem very useful. But I leave that decision to you.

Reply:
I've removed obsolete javadoc from implementation classes.
Thanks.